docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
The command to run the server includes `python manage.py runserver 0.0.0.0:8000`. Ensure that this is suitable for your production environment, as `runserver` is not recommended for production use.